docs/specs/resolve.yml

```diff
@@ -237,7 +237,7 @@ outputs:
   docs/a.json: |
     {"content":"<p>Link to <a href=\"../dep/logo.svg\">dep</a></p>"}
   build.log: |
-    ["warning","link-is-dependency","File 'logo.svg' referenced by link '../dep/logo.svg' will not be built because it is from a dependency docset","docs/a.md"]
+    ["warning","link-is-dependency","File 'logo.svg' referenced by link '../dep/logo.svg' will not be built because it is from a dependency docset","docs/a.md","",1,9]
```

Comment:
why is there empty string for most of the error message?

Reply:
It is `jsonpath`, we could consider turning it into an object

Reply:
let's add a comment or todo :)

Reply:
https://github.com/dotnet/docfx/issues/4141